gulpfile.js

```diff
@@ -223,4 +232,9 @@ gulp.task('watch', function() {
 // This task starts google app engine development server.
 // TODO(Barnabas Makonda): check if files are already generated and if so
 // do not build.
-gulp.task('start_devserver', ['build', 'gulpStartGae', 'watch']);
+var isProdMode = argv.prod_env === 'True';
+if (isProdMode) {
+  gulp.task('start_devserver', ['gulpStartGae', 'watch']);
```

Comment:
Hm, OK -- when you say "this behavior requires hidden knowledge of what some other script is doing" then this is a sign of bad separation of concerns. I think we should fix this, otherwise we're going to get spaghetti code.

Can you find a way to have all the file generation all happen in gulpfile.js, or all of it happen in build.py, but not both? (I think having it all happen in gulpfile.js is probably better.) Consider asking @MAKOSCAFEE, too, if you need help.